products/plesk.md

```diff
@@ -17,6 +17,12 @@ auto:
 
 # eol(x) = releaseDate(x) + 12 weeks
 releases:
+-   releaseCycle: "18.0.55"
+    releaseDate: 2023-08-29
+    eol: 2023-11-23
```

Comment:
```suggestion
    eol: 2023-11-21
```

According to https://www.timeanddate.com/date/dateadded.html?d1=29&m1=08&y1=2023&type=add&ay=&am=&aw=12&ad=&rec=.